File: FizzBuzz/FizzBuzz/FizzBuzzConverter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FizzBuzz
{
    public class FizzBuzzConverter
    {

        public List<String> Convert(List<int> input)
        {
            List<String> result = new List<String>();

            foreach (int i in input)
            {

                string convertString = String.Empty;
                if (IsFizz(i)) { 
                    convertString += "Fizz"; 
                }
                if (IsBuzz(i)) {
                    convertString += "Buzz";
                }
                if (convertString == String.Empty)
                {
                    convertString = i.ToString();
                }
                result.Add(convertString);
            }

            return result;
        }

        public bool IsFizz(int input)
        {
            return (input % 3 == 0);
        }
        public bool IsBuzz(int input)
        {
            return (input % 5 == 0);
        }
    }
}

File: clean-code/src/main/java/ch/adesso/educamp/FizzBuzzConverter.java
package ch.adesso.educamp;

public class FizzBuzzConverter {

    private static final String FIZZ = "Fizz";
    private static final String BUZZ = "Buzz";

    static String[] convert(String[] numbersInRange) {
        for (int i = 1; i <= 100; i++) {
            if (i % 3 == 0 && i % 5 == 0) {
                numbersInRange[i-1] = FIZZ + BUZZ;
            } else if (i % 3 == 0) {
                numbersInRange[i-1] = FIZZ;
            } else if (i % 5 == 0) {
                numbersInRange[i-1] = BUZZ;
            }
        }
        return numbersInRange;
    }

}
